scss/utilities/_overflow.scss

```diff
@@ -0,0 +1,16 @@
+// stylelint-disable scss/dollar-variable-default
+$directions: x, y;
+$name-map: (
+  hide: hidden,
```

Comment:
Make this one `hidden: hidden` so that it's 1:1 with the property-value pair.

Reply:
Noted.